object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,15 @@
+/* exported omit */
+/*  Go through each property found in input source object
+    Check if each found property includes any of the input keys -use includes() method to simply matching input keys to found property
+    If it does not match any of the input keys, push it into an empty storage object
+    Return the filled storage object
+   */
+function omit(source, keys) {
+  var omitted = {};
+  for (var property in source) {
+    if (keys.includes(property) === false) {
```

Comment:
Since `includes` returns a boolean, you could just write `if (!keys.includes(property))`